### DISCUSSION TITLE
Arqiver keeps popping up on GNOME Software app.

### DISCUSSION THREAD

**Discussion by overbeck** [tsujan/Arqiver#74] [Q&A]
Arqiver keeps popping up on the GNOME Software app prompting me to install. Whenever I close it within less than a minute it pops up again. This is really annoying and I want this to stop. Any suggestions would be appreciated.

**Comment by tsujan** [comment]
No suggestion here. The only thing I'm sure of is that it has nothing to do with Arqiver itself. You might want to ask this question in forums related to GNOME or your distro.
